src/js/WhatsappWebWithMoreAccessibility.js

```diff
@@ -1450,7 +1457,7 @@ const activateContextMenu = function (msg) {
             ("pressionou");
             msg.querySelector('[data-testid="down-context"]') ? msg.querySelector('[data-testid="down-context"]').click() : null;
             setTimeout(() => {
-                document.querySelector('[data-testid="react-to-message"]') ? document.querySelector('[data-testid="react-to-message"]').click() : null;
+                document.querySelector('[aria-label="Reagir"]') ? document.querySelector('[aria-label="Reagir"]').click() : null;
```

Comment:
Could you make sure that it will work in other language like English? This way that works just in Portuguese language, but people from other countries also use this. Try to use some id or another CSS selector.

Reply:
ok, i will check that

Reply:
i fixed this issue, also applied the fix that i made in anothe pull request, so i will abort the other one. every change will be in the same pull request